README.md

```diff
@@ -1,39 +1,82 @@
+- **Built in React for React.** We built them in React for React frontends. No cross-framework abstractions and no unnecessary baggage.
 
-Each component in the Propel UI Kit is connected to one metric and can automatically fetch that metric's data, making it easy for users to create visualizations without having to write custom code. Additionally, the components come with a variety of props that can be used to configure queries based on our [Propel GraphQL API](https://propeldata.com/docs/api/about-the-graphql-api). This enables users to customize the components to suit their specific needs.
+- **They work with or without Propel's APIs.** You can use them by themselves in static mode or with the Propel API in connected mode.
 
-Another advantage of the Propel UI Kit is that users can apply their own styles and custom error messages, giving them complete control over the look and feel of their data visualizations.
+## Key features
 
-## Getting Started
+All components share the same set of key features:
 
-To leverage the full power of Propel, you'll need a Propel account and a metric to work with. If you don't have an account yet, you can follow the instructions in [this article](https://propeldata.com/docs/metrics) to get started.
+- **They are responsive.** Components are responsive out of the box. They render well on desktops and laptops as well as phones and tablets.
 
-Once you have your account set up, you'll need to provide your account's access token to use Propel APIs. You can authenticate and generate an access token by following the steps outlined in [this article](https://propeldata.com/docs/api/authentication#step-2-generate-an-access-token).
+- **They handle loading and error states for you.** No need to sweat the details of handling the different states. We've done that for you.
 
-With your metric and access token in hand, it's time to start using Propel UI Kit. Currently, the UI Kit is available for React only and supports the following metric components:
+- **They fetch the data for you.** Components can automatically fetch the data from the Propel API when used in connected mode.
+
+Individual components have other key features that we highlight on the component documentation pages.
+
+## Components
+
+Check out the different components and their documentation:
 
 - [Time Series](https://github.com/propeldata/ui-kit/tree/main/packages/react/time-series)
 - [Counter](https://github.com/propeldata/ui-kit/tree/main/packages/react/counter)
 - [Leaderboard](https://github.com/propeldata/ui-kit/tree/main/packages/react/leaderboard)
 
-Now let's understand how you can integrate Propel UI Kit with your front-end application.
+Please [open a feature request](https://github.com/propeldata/ui-kit/issues) on GitHub for other components you'd like to have. Also, PRs are welcome 😊.
```

Comment:
It's nice to link to a CONTRIBUTING.md here, like we've done in [propeldata/terraform-provider-popel](https://github.com/propeldata/terraform-provider-propel/blob/main/CONTRIBUTING.md).

Reply:
Added a ticket to track this